.github/workflows/main/installDependencies.py

```diff
@@ -81,14 +84,28 @@
 archiveFileName, headers = urlretrieve( args.archiveURL )
 
 os.makedirs( args.dependenciesDir )
-os.system( "tar xf %s -C %s --strip-components=1" % ( archiveFileName, args.dependenciesDir ) )
+if platform != "windows":
+	os.system( "tar xf %s -C %s --strip-components=1" % ( archiveFileName, args.dependenciesDir ) )
+else:
+	subprocess.check_output( "7z x %s -o%s -aoa -y" % ( archiveFileName, args.dependenciesDir ) )
+	# 7z (and zip extractors generally) don't have an equivalent of --strip-components=1
+	# Copy the files up one directory level to compensate
+	for p in glob.glob(
+		os.path.join(
+			args.dependenciesDir.replace( "/", "\\" ),
+			os.path.splitext( args.archiveURL.split( "/" )[-1] )[0],
+			"*"
+		)
+	):
+		shutil.move( p, args.dependenciesDir )
```

Comment:
It's not supported out-of-the-box, so you have to install a program to support `tar.gz` like 7zip that we use here (and is included on the Github workflows images). I think it may be worth sticking with zip as I think lot of Windows-native people would look at the dependencies archive and wonder why they're packaged in a "Linux" format, even though the filename would indicate they're for Windows.